spec/support/job_config.rb

```diff
@@ -3,6 +3,10 @@
     next unless RSpec.current_example.metadata[:type] == :job
     ActiveJob::Base.queue_adapter = :test
     allow(ActiveJob::Base.logger).to receive(:info) # keep the default logging quiet
-    Resque.redis.redis.flushall # clear queues and locks
+    begin
+      Resque.redis.redis.flushall # clear queues and locks
+    rescue Redis::CannotConnectError
+      p "we are rescuing!"
```

Comment:
this is not really a useful message ... but since it is just a spec, probably not worth addressing???

Reply:
It would please me if you changed it;  a quick PR and I'll merge it fast!